openadapt/ml/openai_finetune.py

```diff
@@ -0,0 +1,121 @@
+from openadapt.crud import *
+from openadapt import models, utils
+from copy import deepcopy
+from loguru import logger
+import json
+import openai
+import fire
+
+
+def main(recording_id: int):
+    write_to_file(recording_id)
+
+
+def condense_window_state(recording_id: int):
+    """
+    Takes in a recording as input and outputs a list of tuples
+    of action, window events where window events are condensed
+    to contain basic information, as in stateful_test.py
+    """
+    grab_recording = get_recording_by_id(recording_id)
+    total_acx = get_action_events(grab_recording)
+
+    for i in range(len(total_acx)):
+        processed_acx, processed_wd = (
+            sanitize(total_acx[i])[0],
+            sanitize(total_acx[i])[1],
+        )
+        processed_wd.pop("meta")
+        # task_description = recording.task_description
+
+        logger.debug(f"{total_acx[i].timestamp=}")
+        logger.debug(f"{total_acx[i].window_event_timestamp=}")
+        # which timestamp to finetune on?
```

Comment:
I think for now we should explicitly remove timestamps from fine tuning, with a TODO to consider them